src-ui/src/app/components/common/profile-edit-dialog/profile-edit-dialog.component.html

```diff
@@ -54,7 +54,18 @@ <h4 class="modal-title" id="modal-basic-title" i18n>Edit Profile</h4>
           <p i18n>Connected social accounts</p>
           <ul class="list-group">
             @for (account of socialAccounts; track account.id) {
-              <li class="list-group-item">{{account.name}} ({{account.provider}})<button type="button" class="btn btn-outline-danger btn-sm ms-2 align-baseline" (click)="disconnectSocialAccount(account.id)" i18n-title title="Disconnect {{ account.name }} social account">
+              <li class="list-group-item"
+                ngbPopover="Set a password before disconnecting social account."
+                i18n-ngbPopover
+                [disablePopover]="hasUsablePassword"
+                triggers="mouseenter:mouseleave">
+                {{account.name}} ({{account.provider}})
+                <button
+                  type="button"
+                  class="btn btn-outline-danger btn-sm ms-2 align-baseline"
+                  [disabled]="!hasUsablePassword"
```

Comment:
We could just disable the button if only one social account is remaining and no password is set. Do we also want to prevent the disconnect on the backend in that case?

```
[disabled]="!hasUsablePassword && socialAccounts.length == 1"
```

Reply:
Good idea.

I dont think so, this is just a helpful warning via the UI.

Reply:
Ok, thanks for the quick change with the length